src/App.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Layout from "pages/Layout";
 import Home from "pages/Home";
 import NoPage from "pages/NoPage";
+import NFTs from "pages/NFTs/NFTs";
```

Comment:
Why `pages/NFTs/NFTs`?
I think it is better `pages/NFTs/index.js`
???

Reply:
I changed component name.